release-notes-4.0.0.md

```diff
@@ -15,6 +15,11 @@ ${version-number}
   - The names used for the `configurationProperties` now have a direct mapping to the environment variables used to populate them. For
   example, if a configuration property is named `TASK_PIPE` in a job type yaml definition, then the value for that configuration property
  is expected to be available in an environment variable named `TASK_PIPE`.
+- SCMOD-12730: The `statusCheckUrl` now points to the `status` endpoint instead of the `isActive` endpoint.
```

Comment:
You can include that detail if you want but make the headline "SCMOD-12730: Workers built using framework versions prior to 5.0.0 are no longer supported."